pages/base.py

```diff
@@ -50,30 +50,32 @@ class BasePage(BaseElement):
     def open(self):
         self.driver.get(self.url)
 
+    def verify_page(self):
+        """
+        Use an element from the locator_dictionary to wait for the page to load,
+        then checks if browser is at expected URL.
+        """
+        # self.__getattr__(list(self.locator_dictionary.keys())[0])
+        return self.url.rstrip('/') == self.driver.current_url.rstrip('/')
+
+    def reload(self):
+        self.driver.refresh()
+
 class OSFBasePage(BasePage):
     url = settings.OSF_HOME
 
     def __init__(self, driver):
         super(OSFBasePage, self).__init__(driver)
         self.navbar = self.Navbar(driver)
 
-    locator_dictionary = {
-        'sign_in_button': (By.LINK_TEXT, 'Sign In')
-    }
-
-    def is_logged_in(self):
+    def logged_in(self):
```

Comment:
is_logged_in looks better than the new one.

Reply:
I was debating. Will change the name back. Thanks!